src/commands/check.rs

```diff
@@ -27,7 +31,16 @@ impl CheckCmd {
     fn inner_run(&self) -> Result<()> {
         let config = RUSTIC_APP.config();
         let repo = open_repository(&config.repository)?;
-        repo.check(self.opts)?;
+
+        let groups = repo.get_snapshot_group(&self.ids, SnapshotGroupCriterion::new(), |sn| {
```

Comment:
I think there is also get_matching_snapshots which is better here?!?

Reply:
Pushed some changes, did you mean it like that?

Reply:
Uh, no. I realize that we have `self.ids` - this must be why I used `get_snapshot_group`... I think then it is best to let it as it was... sorry for leading into the wrong direction..

Reply:
No problem ;-)